pkg/apis/serving/metadata_validation_test.go

```diff
@@ -160,11 +162,68 @@ func TestValidateObjectMetadata(t *testing.T) {
 			},
 		},
 		expectErr: (*apis.FieldError)(nil),
+	}, {
+		name: "revision initial scale not parseable",
+		ctx:  config.ToContext(context.Background(), &config.Config{Autoscaler: &autoscalerconfig.Config{AllowZeroInitialScale: true}}),
+		objectMeta: &metav1.ObjectMeta{
+			GenerateName: "some-name",
+			Annotations: map[string]string{
+				autoscaling.InitialScaleAnnotationKey: "invalid",
+			},
+		},
+		expectErr: (&apis.FieldError{Message: "", Paths: []string(nil), Details: ""}).Also(
+			(&apis.FieldError{Message: "", Paths: []string(nil), Details: ""}).Also(
```

Comment:
Why do we have those empty errors? We should be able to delete them.

Reply:
This is a result of the chained `.Also`s https://github.com/knative/serving/blob/81781b53e6556d93a4b580a98178ec6c560901c3/pkg/apis/autoscaling/annotation_validation.go#L45 If we have a test case for error in validating metric annotation and class annotation, we will also see the empty errors (but one less).

Reply:
I just removed the empty errors here: https://github.com/knative/serving/pull/7752
They are completely redundant and unnecessary. FieldError code removes them anyway.

Reply:
Ah i see.

Reply:
I still see the empty errors. Did you push?

Reply:
🤦‍♀️ I missed those. they are updated now